internal/uvm/scsi/mount.go

```diff
@@ -44,6 +44,13 @@ type MountConfig struct {
 	Encrypted bool
 	Verity    *guestresource.DeviceVerityInfo
 	Options   []string
+	// EnsureFilesystem indicates if we should check that a mount has the
+	// target `Filesystem` FS formatted.
+	// This is only supported for LCOW.
+	EnsureFileystem bool
+	// Filesystem is the target filesystem a mount should be mounted as.
```

Comment:
Maybe clearer as `Filesystem is the target filesystem that the device will be mounted as.`

Reply:
Updated.